samples/bluetooth/hap_ha/src/bap_unicast_sr.c

```diff
@@ -223,13 +223,13 @@ static int lc3_reconfig(struct bt_audio_stream *stream, enum bt_audio_dir dir,
 	return -ENOEXEC;
 }
 
-static int lc3_qos(struct bt_audio_stream *stream, const struct bt_codec_qos *qos)
+static struct bt_ascs_rsp lc3_qos(struct bt_audio_stream *stream, const struct bt_codec_qos *qos)
 {
 	printk("QoS: stream %p qos %p\n", stream, qos);
 
 	print_qos(qos);
 
-	return 0;
+	return BT_AUDIO_ASCS_RSP(0, 0);
```

Comment:
I guess if we want to expose `bt_ascs_rsp` as public return value, then we also need to expose the possible values in the API